compiler/src/dashboard_compiler/controls/compile.py

```
@@ -138,6 +138,14 @@ def compile_esql_static_control(order: int, *, control: ESQLStaticValuesControl)
     """
     stable_id = get_layer_id(control)
 
+    # Convert default_value to selectedOptions list
+    selected_options: list[str] = []
+    if control.default_value is not None:
+        if isinstance(control.default_value, str):
+            selected_options = [control.default_value]
+        else:
+            selected_options = control.default_value
```

Ruff (0.14.10)

Use ternary operator `selected_options = [control.default_value] if isinstance(control.default_value, str) else control.default_value` instead of `if`-`else`-block

Replace `if`-`else`-block with `selected_options = [control.default_value] if isinstance(control.default_value, str) else control.default_value`

(SIM108)